addons/delivery/models/delivery_grid.py

```diff
@@ -97,6 +97,8 @@ def _get_price_available(self, order):
                 continue
             if line.product_id.type == "service":
                 continue
+            if not line._is_eligible_for_total():
+                continue
```

Comment:
Why don't you aggregate the two `if` before in the method instead of duplicating the same check ?